cppcache/src/PoolFactory.cpp

```diff
@@ -308,9 +308,15 @@ PoolFactory& PoolFactory::addCheck(const std::string& host, int port) {
   }
 
   ACE_INET_Addr addr(port, host.c_str());
-  if (!(addr.get_ip_address())) {
+
```

Comment:
This is modified, because for IPV6, where ever method get_ip_address() is called, error is reported, and get_addr() must be used. I added this check, to verify is hostname resolved.

Reply:
I think just checking that `get_addr()` is not `nullptr` should suffice.

Reply:
Tried this proposal, and in test, in case hostname is not resolved, get_addr() returns pointer (it is not nullptr).